### DISCUSSION TITLE
How to handle duplicated files after AI noise reduction

### DISCUSSION THREAD

**Discussion by dominictinner** [rcloran/lr-inaturalist-publish#8] [General]
I ran into the problem where I uploaded a rough edit of an observation to iNaturalist and only later started developing it carefully. Usually I use the new AI noise reduction, that creates a new raw file. Is there a way to replace the photo on inat afterwards?
I tried uploading it manually on the website as a second image and then synchronizing again, but this doesn't seem to work (probably because it's basically the same image).

**Comment by rcloran** [comment]
Yes! If you add the observation metadata to the "new" photo (select the old and new photos, then File -> Plug-in Extras -> Group photos into observation), add it to the Observations collection, and remove the old photo from the Observations collection and then "Publish", the observation will be updated with the new photo. Please let me know if that description isn't clear :)
